app.js

```diff
@@ -55,6 +60,23 @@ app.use(function (err, req, res, next) {
   res.status(500).send('Something broke!')
 })
 
-app.listen(port, () => {
-  console.log(`Chat app is listening at http://localhost:${port}`)
-})
\ No newline at end of file
+// Socket IO connection
+io.on('connection', (socket) => {
+  // Access session information
+  let cookieString = socket.request.headers.cookie;
+  let req = { headers : {cookie : cookieString} }
+  cookieSession({ keys: ['superdupersecret'] })(req, {}, function(){})
```

Comment:
You can use an arrow function here, `() => {}`

Reply:
Will address in the next PR.